rediscluster/connection.py

```diff
@@ -383,6 +382,10 @@ def get_node_by_slot(self, slot, *args, **kwargs):
         """
         """
         return self.get_master_node_by_slot(slot)
+    
+    @property
+    def _created_connections(self):
+        return sum(self._created_connections_per_node.values())
```

Comment:
you might want to use self.count_all_num_connections() instead.

Reply:
No, I meant the function `count_all_num_connections` already exists rather than creating a new property that returns the same value.

Reply:
OK I got it. Sorry for the misunderstood. I'll close this PR.